schemas/oval/5.11.3/oval-definitions-schema.xsd

```diff
@@ -1553,13 +1553,8 @@
             <xsd:extension base="xsd:anySimpleType">
                 <xsd:attribute name="name" use="required">
                     <xsd:annotation>
-                        <xsd:documentation>A string restricted to disallow upper case characters.</xsd:documentation>
```

Comment:
Alternative is adding `CaseSensitiveField`. Also the change will relax requirements, it is definitely backward-compatible. I hope it could be possible. Or we will add `YamlField` type or something.